### PR TITLE
MAIN: versions 1.0.0 for all templates

### PR DESCRIPTION
**Pull request by MonikaFeigler**

### CODE REVIEW

**Comment by MonikaFeigler** [comment]
based on this feedback 
https://keboolaglobal.slack.com/archives/C02C3GD6Q64/p1664300490576169

**Comment by MonikaFeigler** [comment]
@webrouse wont you mind to give me a review on this PR, please? It's only about the change of templates versions to 1.0.0 in repository.json. Thank you.

File: .keboola/repository.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
       "path": "shopify-to-datastudio",
       "versions": [
         {
-          "version": "0.0.1",
+          "version": "1.0.0",
```
Comment (webrouse) [inline .keboola/repository.json:15]: Please, fix also `paths`:
image

**Comment by webrouse** [comment]
Paths can be easily fixed:
- by a `regexp` for files content
- by a bulk rename tool for paths itself

File: .keboola/repository.json (hunk 1 above)
Comment (MonikaFeigler) [reply .keboola/repository.json:15]: I've changed it, can you please take a look? If it's what you meant? Thank you! @webrouse

**Review by webrouse — APPROVED** [review]
LGTM

File: .keboola/repository.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
       "path": "thoughtspot-analytics-googleanalytics",
       "versions": [
         {
-          "version": "1.0.1",
```
Comment (ujovlado) [inline .keboola/repository.json:58]: @MonikaFeigler regarding our Slack discussion - this wasn't needed. :) I hope it won't break anything

Comment (webrouse) [reply .keboola/repository.json:58]: I think it's okay (that all the templates in public release have `1.0.0`). 
It won't cause problems, even if someone has already used `1.0.1`.

**Pull request merged**